packages/app/src/app/pages/Dashboard/index.tsx

```diff
@@ -107,10 +107,7 @@ export const Dashboard: FunctionComponent = () => {
 
     const searchParams = new URLSearchParams(location.search);
 
-    if (JSON.parse(searchParams.get('new_workspace'))) {
```

Comment:
I didn't get how this change is related to this scope, is this intentional?

Reply:
Yes. Before this change, the flow for creating a workspace was: stripe / name workspace / invite collaborators / import repos.

But now the workspace is created as a free one again, so: name / invite / import.

This param was used to trigger the workspace rename/invite/import flow after subscribing to pro, which we don't do now anymore.